PARTNERS.md

```diff
@@ -0,0 +1,49 @@
+# OADP Partner Information
+
+## Important Announcement: Version Support Changes
+Starting in 2024, OADP will implement a streamlined version support policy. Red Hat will support only one version of OADP per OpenShift version to ensure better stability and maintainability.
```

Comment:
I suspect this may be the case for MTC until such time as they're aged out after this [proposed] change.  MTC 1.8 already follows along with the supported versions of OADP, so it shouldn't be a concern there.  The concern here is specifically MTC 1.7's dependency on OADP 1.0 in support of legacy migrations from v3.11 to <=v4.16 clusters.  Currently, 1.7 is expected to remain available/supported until 2025-JUL according to [.../support/policy/...](https://access.redhat.com/support/policy/updates/openshift_operators#platform-agnostic):
![image](https://github.com/user-attachments/assets/4f2aebe6-b368-4deb-91f0-cd301d6065b7)

Reply:
Any such existing relationship with other/partner products to consider?

Reply:
I think the `version` must be clearly defined. One `major version`, there are still updates allowed. I know this is mostly understandable by everyone, but needs to be stated.